renovate.json5

```diff
@@ -11,11 +11,12 @@
       "automerge": true,
     },
     {
-      "description": "Compose compiler is tightly coupled to Kotlin version.",
-      "groupName": "Kotlin and Compose",
+      "description": "Compiler plugins are tightly coupled to Kotlin version.",
+      "groupName": "Kotlin",
       "matchPackagePrefixes": [
         "org.jetbrains.kotlin:",
-        "com.google.devtools.ksp"
+        "com.google.devtools.ksp",
+        "dev.drewhamilton.poko",
```

Comment:
Makes sense, my only concern here is the implied (and perhaps unrealistic) expectation that releases of Kotlin and Poko will be in relative lockstep with each other.  Thoughts?

Reply:
This doesn't inherently require that: Renovate will open a PR if either has a new version. This just means that if both have a new version, they'll land in the same PR—which you'll often need to get a successful compilation.

In any case, I think I'm pretty good about timely releases. I do long-running branches for Kotlin betas/RCs to try to catch potential issues early, and usually have the Poko release a couple days after the Kotlin release, if one is required. It did take me about a week when 2.1.0 dropped on Thanksgiving though 😅